app/src/test/java/org/simple/clinic/editpatient/PatientEditScreenControllerTest.kt

```diff
@@ -782,4 +786,26 @@ class PatientEditScreenControllerTest {
     verify(screen, times(2)).showDatePatternInDateOfBirthLabel()
     verify(screen).hideDatePatternInDateOfBirthLabel()
   }
+
+  @Test
+  fun `date-of-birth and age fields should only be visible while one of them is empty`() {
+    uiEvents.onNext(PatientEditAgeTextChanged(""))
+    uiEvents.onNext(PatientEditDateOfBirthTextChanged(""))
+    verify(screen).setDateOfBirthAndAgeVisibility(DateOfBirthAndAgeVisibility.BOTH_VISIBLE)
+
+    uiEvents.onNext(PatientEditDateOfBirthTextChanged("1"))
+    verify(screen).setDateOfBirthAndAgeVisibility(DateOfBirthAndAgeVisibility.DATE_OF_BIRTH_VISIBLE)
```

Comment:
Another place where these enums can be statically imported.

Reply:
Done.